Gopkg.toml

```diff
@@ -65,3 +65,7 @@ version="v1.4.7"
   [[prune.project]]
   name = "sigs.k8s.io/cluster-api"
   unused-packages = false
+
+[[constraint]]
+  name = "github.com/aws/aws-sdk-go"
+  version = "1.17.5"
```

Comment:
Any reason for the specific constrain here rather than just `dep ensure -update github.com/aws/aws-sdk-go`?

Reply:
I'm not sure, but I believe @rudoi needed a newer version for the 

We can remove the implicit `^` constraint for something more open ended like `>=`, if that'd be your preference?

Reply:
I don't recall manually adding this constraint, but git blame doesn't lie. 🤔

Reply:
@sethp-nr >= would be good